src/components/Icons/GoogleMapsIcon.jsx

```diff
@@ -0,0 +1,28 @@
+import { createIcon, extendTheme } from '@chakra-ui/react';
+
+const GoogleMapsIcon = createIcon({
+    displayName: 'GoogleMapsIcon',
+    viewBox: '0 0 15 20',
+    width: '15',
+    height: '20',
+    fill: 'none',
+    xmlns: 'http://www.w3.org/2000/svg',
+
+    path: (
+        <g clipPath="url(#clip0_542_3209)">
+        <path d="M4.57422 14.7685C5.14865 15.5006 5.67146 16.2717 6.13886 17.0763C6.53794 17.833 6.7043 18.3461 6.99595 19.2578C7.17481 19.7616 7.33647 19.9119 7.68402 19.9119C8.06275 19.9119 8.23458 19.6561 8.36733 19.2606C8.64302 18.3996 8.85933 17.7424 9.20061 17.1215C9.87028 15.9228 10.7024 14.8576 11.5201 13.8338C11.7414 13.544 13.1725 11.856 13.8168 10.5241C13.8168 10.5241 14.6087 9.06094 14.6087 7.01756C14.6087 5.10617 13.8278 3.78052 13.8278 3.78052L11.5794 4.38265L10.2139 7.97893L9.87613 8.47483L9.80859 8.56466L9.71875 8.67712L9.56104 8.85675L9.33569 9.08205L8.11979 10.0726L5.07991 11.8278L4.57422 14.7685Z" fill="#34A853"/>
+        <path d="M1.43652 10.2783C2.17845 11.9729 3.60914 13.4626 4.57712 14.7699L9.71851 8.67967C9.71851 8.67967 8.99418 9.62697 7.68027 9.62697C6.21672 9.62697 5.03438 8.45827 5.03438 6.9846C5.03438 5.97403 5.64235 5.27979 5.64235 5.27979L2.15225 6.21497L1.43652 10.2783Z" fill="#FBBC04"/>
+        <path d="M9.77519 0.368652C11.4828 0.919208 12.9443 2.07503 13.8283 3.77941L9.71934 8.67602C9.71934 8.67602 10.3273 7.96922 10.3273 6.96493C10.3273 5.45695 9.05751 4.3304 7.68575 4.3304C6.38856 4.3304 5.64355 5.27614 5.64355 5.27614V2.19134L9.77519 0.368652Z" fill="#4285F4"/>
+        <path d="M2.37793 2.53219C3.39781 1.31196 5.1924 0.0554199 7.66652 0.0554199C8.86689 0.0554199 9.77122 0.370557 9.77122 0.370557L5.63997 5.2788H2.71293L2.37793 2.53219Z" fill="#1A73E8"/>
+        <path d="M1.43646 10.2782C1.43646 10.2782 0.753906 8.94152 0.753906 7.00513C0.753906 5.17498 1.46536 3.57521 2.3783 2.53223L5.64267 5.27927L1.43646 10.2782Z" fill="#EA4335"/>
+        </g>
+    ),
+});
+
+const theme = extendTheme({
+  icons: {
+    GoogleMapsIcon
+  },
+});
+
+export { theme, GoogleMapsIcon };
```

Comment:
⚠️ **[eslint]** <react-refresh/only-export-components> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Fast refresh only works when a file only exports components. Use a new file to share constants or functions between components.